src/components/fields/ArrayField.js

```diff
@@ -50,6 +50,7 @@ function DefaultArrayItem(props) {
     <div key={props.index} className={props.className}>
 
       <div className={props.hasToolbar ? "col-xs-9" : "col-xs-12"}>
+        {props.selectWidget}
```

Comment:
It seems kind of strange for `selectWidget` to be part of the props, and not part of the template.